<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>661</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>320</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Text 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Text 2</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>290</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RA</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastrar">
    <property name="title">
     <string>Cadastrar</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUniversidade"/>
    <addaction name="actionDepartamento"/>
    <addaction name="actionDisciplina"/>
    <addaction name="actionAluno"/>
   </widget>
   <widget class="QMenu" name="menuListar">
    <property name="title">
     <string>Listar</string>
    </property>
    <addaction name="actionTudo"/>
    <addaction name="actionUniversidades"/>
    <addaction name="actionDepartamentos"/>
    <addaction name="actionDisciplina_2"/>
    <addaction name="actionAlunos"/>
   </widget>
   <widget class="QMenu" name="menuSalvar">
    <property name="title">
     <string>Salvar</string>
    </property>
    <addaction name="actionTudo_2"/>
   </widget>
   <widget class="QMenu" name="menuRecuperar">
    <property name="title">
     <string>Recuperar</string>
    </property>
    <addaction name="actionTudo_3"/>
   </widget>
   <addaction name="menuCadastrar"/>
   <addaction name="menuListar"/>
   <addaction name="menuSalvar"/>
   <addaction name="menuRecuperar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUniversidade">
   <property name="text">
    <string>Universidade</string>
   </property>
  </action>
  <action name="actionDepartamento">
   <property name="text">
    <string>Departamento</string>
   </property>
  </action>
  <action name="actionDisciplina">
   <property name="text">
    <string>Disciplina</string>
   </property>
  </action>
  <action name="actionAluno">
   <property name="text">
    <string>Aluno</string>
   </property>
  </action>
  <action name="actionTudo">
   <property name="text">
    <string>Tudo</string>
   </property>
  </action>
  <action name="actionUniversidades">
   <property name="text">
    <string>Universidades</string>
   </property>
  </action>
  <action name="actionDepartamentos">
   <property name="text">
    <string>Departamentos</string>
   </property>
  </action>
  <action name="actionDisciplina_2">
   <property name="text">
    <string>Disciplinas</string>
   </property>
  </action>
  <action name="actionAlunos">
   <property name="text">
    <string>Alunos</string>
   </property>
  </action>
  <action name="actionTudo_2">
   <property name="text">
    <string>Tudo</string>
   </property>
  </action>
  <action name="actionTudo_3">
   <property name="text">
    <string>Tudo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
